test/data/psst-component-data/wrong_psst/psst.json

```diff
@@ -0,0 +1 @@
+broken file content
```

Comment:
https://github.com/brave/brave-core/pull/29044/files#r2089843987 and please fix the ones in this PR